config/awsconfig/awsconfig.go

```diff
@@ -90,6 +91,8 @@ func WithAWS(fs *flag.FlagSet, cb func() (*zap.Logger, bool)) imagor.Option {
 			"Upload ACL for S3 Result Storage")
 		s3ResultStorageExpiration = fs.Duration("s3-result-storage-expiration", 0,
 			"S3 Result Storage expiration duration e.g. 24h. Default no expiration")
+		s3FileStorageClass = fs.String("s3-file-storage-class", "STANDARD",
```

Comment:
better rename to `s3StorageClass`, `s3-storage-class`? As file storage would result confusion with file system storage.

Reply:
should also add storage class to s3 storage, not only result storage. See 
https://github.com/cshum/imagor/pull/402/files#diff-f58e37fdae6c04a8b5d1c70ee04a0c95bbab1a4f55ad66f0ee69796b8fd4730cR165

Reply:
Add test case around https://github.com/cshum/imagor/blob/master/config/awsconfig/awsconfig_test.go
* default should expect `STANDARD`
* a valid setting e.g. `REDUCED_REDUNDANCY`

Reply:
Done.